rebar.config

```diff
@@ -31,3 +31,25 @@
       - (\"diameter_dict_util\":\"format_error\"/\"1\")
       - (\"diameter_dict_util\":\"parse\"/\"2\"))",
          []}]}.
+
+{dialyzer_extra_apps,
+ [
+  diameter
+ ]}.
+
+{dialyzer_warnings,
+ [
+  unmatched_returns,
+  error_handling,
+  race_conditions
```

Comment:
It was added to rebar's set of warnings in 2011 as suggested by @kostis, but if it's a bad idea we can change it later, as it's rebar's own dialyzer settings and therefore doesn't affect the plugin's users.